core/src/test/java/com/taobao/arthas/core/util/ObjectUtilsTest.java

```diff
@@ -0,0 +1,317 @@
+
+    // Act and Assert result
+    Assert.assertEquals(7, ObjectUtils.nullSafeHashCode(array));
+  }
+
+  // Test written by Diffblue Cover.
+  @Test
+  public void nullSafeHashCodeInput0OutputPositive8() {
+
+    // Arrange
+    final float[] array = {};
+
+    // Act and Assert result
+    Assert.assertEquals(7, ObjectUtils.nullSafeHashCode(array));
+  }
+
+  // Test written by Diffblue Cover.
+  @Test
+  public void nullSafeHashCodeInput0OutputPositive9() {
+
+    // Arrange
+    final short[] array = {};
+
+    // Act and Assert result
+    Assert.assertEquals(7, ObjectUtils.nullSafeHashCode(array));
+  }
+
+  // Test written by Diffblue Cover.
+
+  @Test
+  public void nullSafeToStringInput0OutputNotNull() {
+
+    // Arrange
+    final Object[] array = {};
+
+    // Act
+    final String actual = ObjectUtils.nullSafeToString(array);
+
+    // Assert result
+    Assert.assertEquals("{}", actual);
+  }
+
+  // Test written by Diffblue Cover.
+  @Test
+  public void nullSafeToStringInput0OutputNotNull2() {
+
+    // Arrange
+    final byte[] array = {};
+
+    // Act and Assert result
+    Assert.assertEquals("{}", ObjectUtils.nullSafeToString(array));
+  }
+
+  // Test written by Diffblue Cover.
+  @Test
+  public void nullSafeToStringInput0OutputNotNull3() {
+
+    // Arrange
+    final char[] array = {};
+
+    // Act and Assert result
+    Assert.assertEquals("{}", ObjectUtils.nullSafeToString(array));
+  }
+
+  // Test written by Diffblue Cover.
+  @Test
+  public void nullSafeToStringInput0OutputNotNull4() {
+
+    // Arrange
+    final boolean[] array = {};
+
+    // Act and Assert result
+    Assert.assertEquals("{}", ObjectUtils.nullSafeToString(array));
+  }
+
+  // Test written by Diffblue Cover.
+  @Test
+  public void nullSafeToStringInput0OutputNotNull5() {
+
+    // Arrange
+    final int[] array = {};
+
+    // Act and Assert result
+    Assert.assertEquals("{}", ObjectUtils.nullSafeToString(array));
+  }
+
+  // Test written by Diffblue Cover.
+  @Test
+  public void nullSafeToStringInput0OutputNotNull6() {
+
+    // Arrange
+    final short[] array = {};
+
+    // Act and Assert result
+    Assert.assertEquals("{}", ObjectUtils.nullSafeToString(array));
+  }
+
+  // Test written by Diffblue Cover.
+  @Test
+  public void nullSafeToStringInput0OutputNotNull7() {
+
+    // Arrange
+    final long[] array = {};
+
+    // Act and Assert result
+    Assert.assertEquals("{}", ObjectUtils.nullSafeToString(array));
+  }
+
+  // Test written by Diffblue Cover.
+  @Test
+  public void nullSafeToStringInput0OutputNotNull8() {
+
+    // Arrange
+    final float[] array = {};
+
+    // Act and Assert result
+    Assert.assertEquals("{}", ObjectUtils.nullSafeToString(array));
+  }
+
+  // Test written by Diffblue Cover.
+  @Test
+  public void nullSafeToStringInput0OutputNotNull9() {
+
+    // Arrange
+    final double[] array = {};
+
+    // Act and Assert result
+    Assert.assertEquals("{}", ObjectUtils.nullSafeToString(array));
+  }
```

Comment:
It's fine to submit these test but definitely worth creating an enhancement request on these